packages/block-library/src/columns/style.scss

```diff
@@ -28,66 +28,89 @@
 	&.are-vertically-aligned-bottom {
 		align-items: flex-end;
 	}
+
+	&:not(.is-not-stacked-on-mobile) > .wp-block-column {
```

Comment:
What is the benefit of moving these rules inside `&:not(.is-not-stacked-on-mobile) > .wp-block-column`?
These previously were on `.wp-block-column` directly, and it was a lot cleaner. I haven't checked, but would it not be simpler to have these in `.wp-block-column` like before, and then simply override these rules where necessary in `&.is-not-stacked-on-mobile`?
Is there a reason I'm missing for the added complexity?

Reply:
The main benefit is that using the `:not()` rules avoids duplicate code. There are also some `!important` rules here, so we'd end up needing to override those with more `!important` rules. 😅

Reply:
Yes, it's an awkward rule, but results in fewer lines of CSS, while preserving the existing styling / similar specificity. I couldn't quite come up with a neat way to handle it otherwise, so it was a bit of a trade-off! 😅